### PR TITLE
MTCamera & CCCamera implementation

### PR DESCRIPTION
**Pull request by areyesd14**
This PR contains the implementation for MTCamera and their extention for CCCamera (comcam).

### CODE REVIEW

File: love/src/components/MainTel/M2/Actuators/Selector/Selector.jsx [hunk 1]
```diff
@@ -255,6 +255,7 @@ export default class Selector extends Component {
     } = this.props;
 
     const { zoomLevel } = this.state;
+    console.log(zoomLevel);
```
Comment (MiaRoseElbo) [inline love/src/components/MainTel/M2/Actuators/Selector/Selector.jsx:258]: console.log 👀

File: love/src/components/WeatherStation/WeatherStation.module.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 .section {
   display: grid;
-  grid-template-rows: min-content 1fr;
+  grid-template-rows: min-content minmax(auto, 200px);
```
Comment (MiaRoseElbo) [inline love/src/components/WeatherStation/WeatherStation.module.css:14]: Just wondering about the need of this change?

**Review by MiaRoseElbo — COMMENTED** [review]
Looks great! Thanks for the work @areyesd14, I just left like 2 comments regarding very minor stuff.
Thanks again!!

File: love/src/components/WeatherStation/WeatherStation.module.css (hunk 2 above)
Comment (areyesd14) [reply love/src/components/WeatherStation/WeatherStation.module.css:14]: Honestly, I don't know jejeje

File: love/src/components/MainTel/M2/Actuators/Selector/Selector.jsx (hunk 1 above)
Comment (areyesd14) [reply love/src/components/MainTel/M2/Actuators/Selector/Selector.jsx]: Thanks Mia!

**Review by MiaRoseElbo — APPROVED** [review]
Great work!!

**Pull request merged**